test/data/starwars.ttl

```diff
@@ -55,7 +55,7 @@
     :primaryFunction 'Astromech' .
 
 :NewHope a :Episode ;
-	:name "A New Hope" ;
```

Comment:
I don't think it is right changing the name of this classic title. I'd add a new entry so we don't get sued by Disney.

Reply:
I would agree with this sentiment prior to a new standard which was released. When they blew up a *3rd* death star it was made clear that we are to disregard all that is sacred for that which is convenient, and dare I say lazy.

Reply:
I stopped watching when they blew up the second death star, but I guess that doesn't give me the right rename their franchise. I fixed this so I am adding a new entry instead.